src/third_party_libs/header_only/cereal/external/rapidjson/genericstream.h

```diff
@@ -78,7 +78,7 @@ namespace rapidjson {
       }
 
       size_t Tell() const {
-        return (int)os_.tellp();
+        return static_cast<int>(os_.tellp());
```

Comment:
Ah right cool, I thought it was just these ones. I would perhaps do a low priority task for a PR to the cereal team on this issue. But I will merge now. Cheers